.github/actions_scripts/get_release_vars.sh

```diff
@@ -1,7 +1,14 @@
 #!/bin/bash
 
-VERSION=$(cat ${1}/Jamulus.pro | grep -oP 'VERSION = \K\w[^\s\\]*')
-echo "::set-output name=JAMULUS_VERSION::${VERSION}"
-# get the tag which pushed this
-echo "::set-output name=PUSH_TAG::${GITHUB_REF#refs/*/}"
+set -e 
+
+# get version from project-file
+JAMULUS_VERSION=$(cat ${1}/Jamulus.pro | grep -oP 'VERSION = \K\w[^\s\\]*')
+
+# get the tag/branch-name which pushed this
+PUSHED_NAME=${GITHUB_REF#refs/*/}
+
+# calculate various variables
+python3 ${1}/.github/actions_scripts/analyse_git_reference.py ${GITHUB_REF} ${PUSHED_NAME} ${JAMULUS_VERSION} 
```

Comment:
Would it be possible to also get the Jamulus version and Tag via python? this would be much cleaner.

Reply:
possible, but more part of #854 optimizations.